docs/NNNN-flat-direct-clean-storage-layout.md

```diff
@@ -0,0 +1,64 @@
+# OCFL Community Extension NNNN: Flat Direct Clean Storage Layout
+
+* **Extension Name:** NNNN-flat-direct-clean-storage-layout
+* **Authors:** Jürgen Enge, Basel
+* **Minimum OCFL Version:** 1.0
+* **OCFL Community Extensions Version:** 1.0
+* **Obsoletes:** n/a
+* **Obsoleted by:** n/a
+
+## Overview
+
+This object/storage root extension maps content file names or OCFL object identifiers by replacing "dangerous characters" from file and folder names.
```

Comment:
OCFL layout extensions are designed to map OCFL object identifiers to storage layout paths. I am not sure I see where "content file names" come into play.

Reply:
This Extension can be used as an object extension to clean content path names. It's not only intended for identifiers.
Therefor it can reside in the storage root extensions folder as well as in the extensions folder of the base directory of an object.
Should there be a different name for the same functionality when used as object extension?